lib/generate_letters.ml

```diff
@@ -78,10 +78,24 @@ let refill_deck deck =
   in
   helper deck (7 - List.length deck)
 
+let rec has_vow = function
+  | [] -> false
+  | h :: t ->
+      if List.mem h [ 'A'; 'E'; 'I'; 'O'; 'U' ] then true else has_vow t
+
+let insert_vow lst =
+  match lst with
+  | [] -> []
+  | _ :: t -> get_random_letter vowel_amounts :: t
+
+let ensure_vowel lst = if has_vow lst then lst else insert_vow lst
```

Comment:
nit: be consistent w/ naming, `ensure_vowel` is good but then the helper functions should be called `has_vowel` and `insert_vowel`